eclipse/src/saros/project/SharedResourcesManager.java

```diff
@@ -90,17 +91,25 @@
       new ISessionListener() {
 
         @Override
-        public void projectAdded(saros.filesystem.IProject project) {
+        public void projectAdded(saros.filesystem.IReferencePoint referencePoint) {
           synchronized (projectStates) {
-            IProject eclipseProject = (IProject) ResourceAdapterFactory.convertBack(project);
+            IReferencePointManager referencePointManager =
+                sarosSession.getComponent(IReferencePointManager.class);
+            IProject eclipseProject =
```

Comment:
In my Tests the `referencePointManager` has no mapping and eclipseProject is null here leading to a NPE

Reply:
fixed